docs/assets/diagrams/transfer/05-authorize-transaction.puml

```diff
@@ -0,0 +1,80 @@
+@startuml
+
+title Authorize Transaction
+
+hide footbox
+
+autonumber
+
+box "Mobile Device" #Light
+  actor Alice
+  participant "PISP App" as App
+end box
+box "PISP" #Light
+  database Firebase
+  participant "PISP Server" as Server
+end box
+box "Mojaloop" #Light
+  participant "Switch" as Mojaloop
+end box
+
+Alice -> App: Authorize the transaction
+
+rnote right of App #Light
+  ""firebase.firestore()""
+  ""  .collection("transactions")""
+  ""  .doc("abc123")""
+  ""  .set({""
+  ""    authenticationInfo: {""
+  ""      authentication: "U2F",""
+  ""      authenticationValue: {""
+  ""        pinValue: "base64(xxx)",""
+  ""      }""
+  ""    },""
```

Comment:
Please think about what the optimal representation here is. It sounds to me like this is only represented this way because it's the way Mojaloop does it. But we DO NOT want to be a simple proxy server between a phone and Mojaloop. We need to be an ABSTRACTION layer.

Reply:
Updated the field to the following object representation:
```
authentication: {
  type: "U2F",
  value: "base64(xxx)"
}
```

The rationale behind this change:
- The `type` field is to support flexibility in case we would like to support other authentication methods in the future (e.g., OTP, Web)
- The `value` field is a generic field that can contain any necessary value to perform the authentication (e.g., FIDO signature, OTP code, short-lived token).

Any thoughts on the new authentication fields?